pmm/v3/pmm3-api-tests.groovy

```diff
@@ -85,9 +85,8 @@ pipeline {
                 sh '''
                     docker run -d \
                     -e PMM_DEBUG=1 \
-                    -e PERCONA_TEST_CHECKS_INTERVAL=10s \
```

Comment:
Just one question: do we want to drop this variable?

Reply:
Yeah since it's not used anywhere in PMM atm